### ISSUE TITLE
Idea request. Mech armor

### ISSUE THREAD

**Issue by Mii1128** [Esteemed-Innovation/Flaxbeards-Steam-Power#313]
Basically a Mech suit you can equip with stronger tools and weapon.
- 3x3 Drill
- Whole Tree chopping
- Stronger weapons
- Strong armor
- Jetpack?
Would be expensive to build and use a LOT of steam and be quite slow

**Comment by elifoster** [comment]
I don't understand the purpose of this...

**Comment by Mii1128** [comment]
It would be awesome ... and different than the other mods.

**Comment by elifoster** [comment]
Right, but when we already have the Exosuit and the upcoming Steam Tool modifications I don't think it really has much use.

**Comment by Divineaspect** [comment]
um didn't I see someone we know showing gifs of his giant armor effect, we could impliment something like that to enhance the shitkicking feel on top of the exosuit.

**Comment by elifoster** [comment]
That was @Flaxbeard that you are thinking about.
